x-pack/plugins/cases/public/components/create/custom_fields.tsx

```diff
@@ -9,16 +9,34 @@ import React, { useMemo } from 'react';
 import { sortBy } from 'lodash';
 import { EuiFlexGroup, EuiFlexItem, EuiSpacer, EuiText } from '@elastic/eui';
 
+import { useFormData } from '@kbn/es-ui-shared-plugin/static/forms/hook_form_lib';
 import type { CasesConfigurationUI } from '../../../common/ui';
 import { builderMap as customFieldsBuilderMap } from '../custom_fields/builder';
 import * as i18n from './translations';
+import { useCasesContext } from '../cases_context/use_cases_context';
+import { useGetAllCaseConfigurations } from '../../containers/configure/use_get_all_case_configurations';
+import { getConfigurationByOwner } from '../../containers/configure/utils';
 
 interface Props {
   isLoading: boolean;
-  customFieldsConfiguration: CasesConfigurationUI['customFields'];
 }
 
-const CustomFieldsComponent: React.FC<Props> = ({ isLoading, customFieldsConfiguration }) => {
+const CustomFieldsComponent: React.FC<Props> = ({ isLoading }) => {
+  const { owner } = useCasesContext();
+  const [{ selectedOwner }] = useFormData({ watch: ['selectedOwner'] });
+  const { data: configurations, isLoading: isLoadingCaseConfiguration } =
+    useGetAllCaseConfigurations();
+
+  const configurationOwner: string = selectedOwner ? selectedOwner : owner[0];
```

Comment:
`owner[0]` can be `undefined` because you can pass an empty array to the `CasesContextProvider` as an owner.

Reply:
I will change the type here

```suggestion
  const configurationOwner: string | undefined = selectedOwner ? selectedOwner : owner[0];
```

and handle it inside `getConfigurationByOwner` to return the initial configuration.